app/controllers/resource_categories_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ResourceCategoriesController < ApplicationController
+  before_action :find_resource_category, only: [:show, :update, :destroy]
+
+  def show
+    render json: @resource_category
+  end
+
+  def create
+    # TODO: authenticate admin user
```

Comment:
Only admin users in the CMS are going to have the ability to create, update, and destroy, but we don't have a user model or authentication method yet -- coming in a follow-up PR.